### PR TITLE
Unix command sysctl

### PR DESCRIPTION
**Pull request by marcin-usielski**

### CODE REVIEW

File: moler/cmd/unix/sysctl.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+"""
+Ping command module.
```
Comment (tkrol88) [inline moler/cmd/unix/sysctl.py:3]: Ping --> Sysctl

File: moler/cmd/unix/sysctl.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+    def build_command_string(self):
+        """
+        Builds command string from parameters passed to object.
+        :return: String representation of command to send over connection to device.
+        """
+        cmd = "sysctl"
+        if self.options:
+            cmd = "{} {}".format(cmd, self.options)
+        if self.parameter and self.options:
```
Comment (tkrol88) [inline moler/cmd/unix/sysctl.py:46]: hmm, you probably mean <br>
```
if self.parameter and self.value:
```

File: moler/cmd/unix/sysctl.py (hunk 1 above)
Comment (marcin-usielski) [reply moler/cmd/unix/sysctl.py]: Done

File: moler/cmd/unix/sysctl.py (hunk 2 above)
Comment (marcin-usielski) [reply moler/cmd/unix/sysctl.py]: Done

**Pull request merged**